private-templates-service/client/src/components/Designer/styled.tsx

```diff
@@ -4,7 +4,7 @@ export const OuterDesignerWrapper = styled.main`
   display: flex;
   flex-direction: column;
   height: 100%;
-  overflow-behaviour: contain;
+  overscroll-behaviour: contain;
```

Comment:
is it supposed to be behavior?

Reply:
i think so, `overscroll-behavior: contain;` sets the inner elements to be unscrollable.
https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior

Reply:
I mean without the u in behaviour!

Reply:
haha whoops, fixed